<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectFileWindow</class>
 <widget class="QDialog" name="selectFileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonSelectLeft">
      <property name="text">
       <string>Select left half of video</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSelectRight">
      <property name="text">
       <string>Select right half of video</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>291</width>
     <height>204</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelPathLeft">
      <property name="text">
       <string>PathL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPathRight">
      <property name="text">
       <string>PathR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>279</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButtonExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonPlay">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>77</y>
     <width>53</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Frames:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_LeftHalf">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Frames:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_RightHalf">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
